src/nova-driver.md

```diff
@@ -0,0 +1,54 @@
+# Nova GPU Driver
+
+Nova is a driver for GSP-based Nvidia GPUs that is currently under development
+and is being written in Rust.
+
+As of today, the driver is in the (very) early stage and not yet operational.
+
+
+## Contact
+
+To contact the team and / or participate in development, please use the
+mailing list of the nouveau-driver for now: nouveau@lists.freedesktop.org
+
+
+## Resources
+
+- [Official Source Tree](https://gitlab.freedesktop.org/drm/nova)
+- [Announcement E-Mail](https://lore.kernel.org/dri-devel/Zfsj0_tb-0-tNrJy@cassiopeiae/)
+
+In the source tree, the driver lives in `drivers/gpu/drm/nova`
+
+
+## Utilized Common Rust Infrastructure
+
+Currently, Nova is based on those downstream branches:
+
+- rust-common
+- rust-device
+- rust-drm
+- rust-pci
+
+This number will likely decrease over time once more and more necessary Rust
+infrastructure enters the kernel.
+
+
+## Contributing
+
+As with every real open source program, help and participation is highly welcome!
+
+As the driver is very young, however, it is currently difficult to assign tasks
+to people. Many things still have to settle until a steadily paced workflow
+produces atomic work topics a new one can work on.
+
+If you really want to jump in immediately regardless, here are a few things you
+can consider:
+
+- Most work to do right now is with more bindings for Rust. Notably, this includes
+the device driver model, DRM and PCI. If you have expertise there, have a look
+at the existing code and see if there's something you can add or improve.
+- Feel free to go over Nova's code base and make suggestions or send patches,
+for example for improved comments, grammar fixes, improving code readability etc.
```

Comment:
I think that's good as well. But I think we should (somewhere in this document) distinguish between nova and nova-next (previous stub/nova).